pysb/bng.py

```diff
@@ -116,6 +119,11 @@ def __enter__(self):
     def __exit__(self):
         return
 
+    def _delete_tmpdir(self):
+        for f in os.listdir(self.base_directory):
+            os.unlink(os.path.join(self.base_directory, f))
+        os.rmdir(self.base_directory)
+
```

Comment:
If you want to explicitly call unlink on a list of known BNG-produced filenames, then rmdir the parent, I could get behind that too. But that middle ground of reimplementing a one-level-only rmtree seems like the worst of both worlds (*won't* error if other unexpected files somehow end up in the temp dir, and *will* error if a later version of BNG starts doing stuff in subdirectories).